Documentation.md

```diff
@@ -197,6 +197,12 @@ this:
 
 ## `connect.core`
 
+### `connect.core.initialized`
+```js
+if (connect.core.initialized) { /* ... */ }
+```
+Indicates whether the CCP has been initialized.
```

Comment:
I think that this is misleading. connect.core.initialized does not directly correspond to whether the CCP is fully initialized. Rather, it indicates whether Streams has successfully sent the CONFIGURE event to the shared worker. In practice, this means largely nothing outside of Streams' internal implementation. I do not recommend that customers use this for any sort of logic. Agent polling is not yet started when we mark "connect.core.initialized" to true.

Reply:
"Indicates whether Streams communicated succesfully with the CCP" may be an acceptable message here.

Reply:
Make sense, since customers are still using it, I believe it should be documented to warn against its usage. I have update the commit with the following documentation:

> Indicates whether `amazon-connect-streams` communicated successfully with the CCP.
> **Important:** This does not mean that the CCP has been fully initialized, and it's
mostly used internally by `amazon-connect-streams`. It's not recommended that you use
this field to detect whether the CCP has been initialized.
As an alternative, consider using `connect.agent()`.